pkg/target/complexcomposition_test.go

```diff
@@ -227,40 +227,6 @@ resources:
 		err.Error(), "already registered id: apps_v1_StatefulSet|~X|my-sts") {
 		t.Fatalf("Unexpected err: %v", err)
 	}
-
-	// Expected Output
```

Comment:
Why `devMergeResult` and `prdMergeResult` are removed? Are those two tests to be fixed?

Reply:
unused var.

Reply:
How about moving the while var into a comment. Since it's marked as `Expected Output`, I'm concerned that it can be useful for future fix.

Reply:
reverted the file.  will figure out the intent some other time.